src/Service/Route53/tests/Unit/Input/ChangeResourceRecordSetsRequestTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Route53\Tests\Unit\Input;
+
+use AsyncAws\Core\Test\TestCase;
+use AsyncAws\Route53\Enum\ChangeAction;
+use AsyncAws\Route53\Enum\ResourceRecordSetFailover;
+use AsyncAws\Route53\Enum\RRType;
+use AsyncAws\Route53\Input\ChangeResourceRecordSetsRequest;
+use AsyncAws\Route53\ValueObject\Change;
+use AsyncAws\Route53\ValueObject\ChangeBatch;
+use AsyncAws\Route53\ValueObject\ResourceRecord;
+use AsyncAws\Route53\ValueObject\ResourceRecordSet;
+
+class ChangeResourceRecordSetsRequestTest extends TestCase
+{
+    public function testRequest(): void
+    {
+        $input = new ChangeResourceRecordSetsRequest([
+            'HostedZoneId' => 'Z1D633PEXAMPLE',
+            'ChangeBatch' => new ChangeBatch([
+                'Comment' => 'foo',
+                'Changes' => [
+                    new Change([
+                        'Action' => ChangeAction::UPSERT,
+                        'ResourceRecordSet' => new ResourceRecordSet([
+                            'Name' => 'www.example.com',
+                            'Type' => RRType::A,
+                            'SetIdentifier' => 'Main',
+                            'Weight' => 100,
+                            'Failover' => ResourceRecordSetFailover::PRIMARY,
+                            'TTL' => 60,
+                            'ResourceRecords' => [
+                                new ResourceRecord([
+                                    'Value' => '34.145.17.120',
+                                ]),
+                            ],
+                        ]),
+                    ]),
+                ],
+            ]),
+        ]);
+
+        // see example-1.json from SDK
+        $expected = '
+POST /2013-04-01/hostedzone/Z1D633PEXAMPLE/rrset/ HTTP/1.0
+Content-Type: application/xml
+';
+
+        self::assertRequestEqualsHttpRequest($expected, $input->request());
```

Comment:
Is it true? I would expect X:L content in the body... That's weird the test is not failling

Reply:
oh! it looks like the generator is not able to generate code for RestXML  (until now, we only deal with S3 and file's content in BODY

https://github.com/async-aws/aws/blob/e2edb3dd266aa76ef2ab87040209728fe9c655ff/src/CodeGenerator/src/Generator/RequestSerializer/RestXmlSerializer.php#L37-L39

I'll open a PR to fix that.